### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by tkindy**

### CODE REVIEW

**Comment by TomPed** [comment]
What should be do about coveralls?

**Comment by tkindy** [comment]
We can merge without it passing, it isn't one of the required checks.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "america-scores-backend",
-  "version": "0.1.0",
+  "version": "0.2.0",
```
Comment (tkindy) [inline package.json]: Nice call! 👍

File: config/connection.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 const mysql = require('mysql');
 const Promise = require('bluebird');
-const env = process.env.NODE_ENV || 'development';
+const env = 'development';
```
Comment (TomPed) [inline config/connection.js]: Should we figure out this fix for AWS?

File: lib/utils.js [hunk 3]
```diff
@@ -18,15 +18,38 @@ function isPositiveInteger(str) {
...
+QueryError.prototype.constructor = QueryError;
+
 function query(queryString, args) {
   return Promise.using(getSqlConnection(), function(connection) {
     return connection.queryAsync(queryString, args)
-      .then(function(rows) {
-        return rows;
-      });
+    .catch(function(err) {
+      if (err.code == 'ER_TRUNCATED_WRONG_VALUE_FOR_FIELD' ||
```
Comment (tkindy) [inline lib/utils.js]: What's the word with this?

**Pull request merged**